test/e2e_env/universal/envoyconfig/envoyconfig.go

```diff
@@ -50,6 +51,18 @@ func EnvoyConfigTest() {
 			Setup(universal.Cluster)
 		Expect(err).ToNot(HaveOccurred())
 
+		Eventually(func(g Gomega) {
+			spec, status, err := GetMeshServiceStatus(universal.Cluster, "test-server", meshName)
```

Comment:
you have to check both test-server and demo-client

Reply:
added